public/script.js

```diff
@@ -0,0 +1,56 @@
+const sourceTextContainer = document.querySelector('.source-text-container')
+
+const spanArray = []
+
+const h1 = document.querySelector('h1')
+
+function populateSourceText(data) {
+  sourceTextContainer.innerHTML = ''
+  spanArray.splice(0, spanArray.length)
+  
+  data.forEach((text) => {
+    const p = document.createElement('p')
+    text.split(" ").forEach((word) => {
+      const span = document.createElement('span')
+      span.innerText = word
+      span.tabIndex = "4"
+      spanArray.push(span)
+      p.appendChild(span)
+      p.appendChild(document.createTextNode(" "))
+    })
+    sourceTextContainer.appendChild(p)
+  })
+
+  // puts the first word in the title for screen reader
+
+  let firstWord = document.querySelector('#first-word')
+  if (firstWord !== null) {
+    firstWord.remove()
+  }
+  firstWord = document.createElement('span')
+  firstWord.id = 'first-word'
+  firstWord.classList.add('sr-only')
+  firstWord.innerText = 'The first word is: ' + spanArray[0].innerText
+  h1.appendChild(firstWord)
```

Comment:
i also don't really understand why you tell the screen reader user what the first word is, because surely then afterwards they would have to find out somewhere else what the rest of the words are 🤔

Reply:
when the user presses space, the JS is gonna focus then next word in the source text - so if a screen reader is being used, it'll read out that next word. That works for all the words apart from the first one. We would add in the screen reader instructions 'press tab three times to hear the first word' but we thought putting the first word of the text in the instructions would make for a smoother experience